rider-fsharp/src/main/resources/META-INF/plugin.xml

```diff
@@ -84,6 +84,7 @@
     </intentionAction>
 
     <registryKey key="rider.fsharp.experimental" description="Enable experimental F# features" defaultValue="false" restartRequired="false"/>
+    <registryKey key="rider.fsharp.formatter" description="Enable new F# code formatter" defaultValue="false" restartRequired="false"/>
```

Comment:
We didn't have a working R# formatter previously, and Fantomas is doing a somewhat different thing in Reformat Code action, so I'd prefer it without `new` word here. Though, I doubt many people will ever see this temporary setting. :)